doc/api/http.md

```diff
@@ -107,6 +107,12 @@ http.get({
 });
 ```
 
+<!-- YAML
```

Comment:
We usually use only one YAML block per section. See `changes:` fields in other YAML blocks as examples of describing changes.

Reply:
got it, thanks!
my only concern is that if we put the change in that block then it's not immediately visible or searchable because the history block is collapsed.

Reply:
Let's see what others think. Maybe we can double this info in the text of the sections.